warehouse/templates/pages/credits.html

```diff
@@ -0,0 +1,42 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "base.html" %}
+{% block title %}{% trans %}Development Credits{% endtrans %}{% endblock %}
+
+{% set core_contributors = ['ewdurbin', 'di', 'dstufft', 'nlhkabu',
+                        'yeraydiazdiaz'] %}
+
+{% block content %}
+  <section class="horizontal-section">
+  <div class="narrow-container">
+    <h2>{% trans %}Development Credits{% endtrans %}</h2>
+      <p>{% trans %}We'd like to thank the following people who have helped build the{% endtrans %} <a href="https://pypi.org/">{% trans %}PyPI Warehouse{% endtrans %}</a> {% trans %} application for the Python community{% endtrans %}:</p>
```

Comment:
Splitting text in parts like this will not work for many languages.
This could be one text with a little markup in the middle (trusting translator to keep it as is), or you could move the markup string to a variable.